autoname_workspaces.py

```diff
@@ -136,5 +144,9 @@ def window_event_handler(i3, e):
         if e.change in ['new', 'close', 'move']:
             rename_workspaces(i3)
 
+    def workspace_event_handler(i3, e):
+        rename_workspaces(i3)
```

Comment:
You can also try this experimentally by saving this chunk of code into a python file and running it:

~~~{.py}
import i3ipc

def handle(i3, e):
    print(e)
    print(e.change)

i3 = i3ipc.Connection()
i3.on('workspace', handle)
i3.main()
~~~